Game/Resources/Constant Scene/Minecart/Script/Player.as

```diff
@@ -0,0 +1,20 @@
+class Player {
+    Hub @hub;
+    Entity @self;
+    bool first;
+
+    Player(Entity @entity){
+        @hub = Managers();
+        @self = @entity;
+        first = true;
+        // Remove this if updates are not desired.
+        RegisterUpdate();
+    }
+
+    // Called by the engine for each frame.
```

Comment:
Unnecessary comment, since this is true with all scripts.

Reply:
This is not my doing, I will not acquiesce this change. The script stays as is!